src/components/forms/SelectField/SelectField.js

```diff
@@ -0,0 +1,87 @@
+import {FormFieldDescription, FormLabel, Paragraph} from '@utrecht/component-library-react';
+import {Field} from 'formik';
+import PropTypes from 'prop-types';
+import {FormattedMessage} from 'react-intl';
+import Select from 'react-select';
+
+import {getBEMClassName} from 'utils';
+
+const SelectField = ({
+  name,
+  label = '',
+  id = '',
+  isRequired = false,
+  description = '',
+  disabled = false,
+  invalid = false,
+  options = [],
+  valueProperty = 'value',
```

Comment:
I copied the implementation from the form-builder repository, providing a higher-level API to the consumers of these components instead of having to define callback functions everywhere.